src/hooks/Guilds/useUpdateEnsContent.ts

```diff
@@ -1,13 +1,13 @@
 import { useMemo } from 'react';
 import { UpdateENSContentDecodedCall } from 'components/ActionsBuilder/SupportedActions/UpdateENSContent/types';
 
-/*
-  @from: Guild address
-  @to ENS resolver address
-  @node: Namehash of the ENS name 
-  @contentHash: IPFS/CID hash
-  @optionalProps: Props needed to update the ENS frontend content
-  */
+/**
+ * @property {string} from Guild address

```

Comment:
Where are these comments used? Are we planning to setup JSDoc or something?

Reply:
I thought it was the standard to write comments for `useUpdateEnsContent` since the properties are not immediately apparent. But maybe it's better to just stick to TypeScript tbh.

Reply:
Had a discussion on Keybase and decided to keep the comments as-is. Going forward we can start to set a comment standard around `JSDoc?, if the rest of the squad agrees.